modules/rte_mis_lottery/rte_mis_lottery.routing.yml

```diff
@@ -25,3 +25,16 @@ rte_mis_lottery.config.settings:
     _title: 'RTE MIS Lottery Settings'
   requirements:
     _permission: 'administer site configuration'
+
+# RTE MIS Lottery External API.
+rte_mis_lottery.data:
+  path: '/api/v1/lottery-data'
+  defaults:
+    _controller: '\Drupal\rte_mis_lottery\Controller\DataController::handlePost'
```

Comment:
Use the existing controller where we have created status API.

Reply:
ok